src/js/rtc_const.js

```diff
@@ -35,7 +35,7 @@ export const RTC_PEER_CONNECTION_CONFIG = {
     iceTransportPolicy: 'relay',
     rtcpMuxPolicy: 'require',
     bundlePolicy: 'balanced',
-    sdpSemantics: 'unified',
```

Comment:
```
Phase 4: Make “Plan B” throw
In this phase, setting the sdpSemantics flag to "plan-b" results in an exception being thrown. It has been throwing in Canary from M93. As of M96, the exception was throwing in all channels, including Stable.

During this phase, a [Deprecation Trial](https://developer.chrome.com/origintrials/#/view_trial/3892235977954951169) was available that allowed using Plan B without the exception throwing but the trial stopped working on May 25, 2022.
```

Can you please confirm if this change is safe for future chrome release?

Reply:
https://webrtc.org/getting-started/unified-plan-transition-guide#phase_4_make_%E2%80%9Cplan_b%E2%80%9D_throw

Reply:
I believe so. That milestone happened in M99 (Mar/2022): https://groups.google.com/g/discuss-webrtc/c/Yf6c3HW4N3k/m/3SC_Hy15BQAJ . Chrome changed default to "unified-plan" and does not throw an exception if you are trying to set SDP Semantics to that value. The only reason we need to be explicit now (instead of implicitly selecting unified-plan) is because one of the vendor's UC SDK WebRTC API had "plan-b" as the default and we had to force it to use `unified-plan`.

Reply:
Got it. As long as `unified-plan` is not throwing exception in future phases well. Looks good to approve.